postprocessing_variant_calls/maf/helper.py

```diff
@@ -323,38 +336,24 @@ def tag_all(self, tagging):
             self.tag("prevalence_in_cosmicDB")
             self.tag("truncating_mut_in_TSG")
         return self.data_frame
-    
+
     def tag_traceback(self, cols, tagging):
         if set(cols["standard"] + cols["access"]).issubset(
-                set(self.data_frame.columns.tolist())
```

Comment:
unsure why there are changes here? Is this because the PR needs to merge in develop?